src/redux/Categories/Categories.js

```diff
@@ -0,0 +1,16 @@
+export const addCategory = () => ({
+  type: 'CHECK_STATUS',
```

Comment:
- [ ] Again, this CHEK_STATUS action type should be define at the top and must follow the ducks pattern, so it can be something like `const CHECK_STATUS =  bookstore/categories/CHECK_STATUS. ` Here, I tried to scope the action name based on the feature it belongs to.

Reply:
Thank you for your review. I have solved this issue.